### DISCUSSION TITLE
How to change the comfyUI scratch disk ?

### DISCUSSION THREAD

**Discussion by Creative-comfyUI** [comfyanonymous/ComfyUI#4116] [General]
It looks like comfyUI uses the main disc for its operation and not the disc where comfyUI is located. Whatever the case, how to set comfyUI to use a specific disc. Thanks

**Comment by ltdrdata** [comment]
When running ComfyUI from a location other than the main disk, if the main disk is being heavily used, there are two possible scenarios:

1. Using a Python environment configured on the main disk rather than a portable installation
2. Swapping (a.k.a. virtual memory) occurring due to insufficient memory

For case 1, the solution is to install the portable version and run it using only that Python environment.
For case 2, the best solution is to increase physical RAM to prevent swapping from occurring.
